addons/payment_alipay/models/payment_acquirer.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from hashlib import md5
+from werkzeug import urls
+
+from odoo import api, fields, models, _
+from odoo.tools.float_utils import float_compare
+from odoo.addons.payment_alipay.controllers.main import AlipayController
+from odoo.addons.payment.models.payment_acquirer import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    provider = fields.Selection(selection_add=[
+        ('alipay', 'Alipay')
+    ], ondelete={'alipay': 'set default'})
+    alipay_payment_method = fields.Selection([
+        ('express_checkout', 'Express Checkout (only for Chinese Merchant)'),
+        ('standard_checkout', 'Cross-border'),
+    ], string='Account', default='express_checkout',
```

Comment:
The doc is super vague too. Sometimes they name it "Wallet". "Checkout type" maybe? But that's not really better. "Account type" is no good.
But once you want to integrate, the words "Express checkout" or "Cross-border" will ring a bell because that's related to the type of account you create on their platform.

Reply:
Account it is then